hudi-utilities/pom.xml

```diff
@@ -171,6 +171,14 @@
       </exclusions>
     </dependency>
 
+    <!-- Pulsar Spark Connector -->
+    <dependency>
```

Comment:
the bundle needs to change too?

Reply:
yes, wondering how did you happen to test this change w/o the bundle.